K8s-deployment/Charts/latest-ingestion-pipeline/example-aws-resource-values.yaml

```diff
@@ -0,0 +1,79 @@
+## processor resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param processor.resources.limits The resources limits for the processor containers
+## @param processor.resources.requests The requested resources for the processor containers
+##
+processor:
+  resources:
+    limits:
+      cpu: 800m
+      memory: 1Gi
+    requests: 
+      cpu: 600m
+      memory: 800Mi
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+
+## rabbitmq resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param rabbitmq.resources.limits The resources limits for the rabbitmq containers
+## @param rabbitmq.resources.requests The requested resources for the rabbitmq containers
+##
+rabbitmq:
+  resources:
+    limits:
+      cpu: 600m
+      memory: 750Mi
+    requests: 
+      cpu: 400m
+      memory: 600Mi
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+
+## redis resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param redis.resources.limits The resources limits for the redis containers
+## @param redis.resources.requests The requested resources for the redis containers
+##
+redis:
+  resources:
+    limits:
+      cpu: 700m
+      memory: 850Mi
+    requests: 
+      cpu: 400m
+      memory: 700Mi
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+
+## postgres resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param postgres.resources.limits The resources limits for the postgres containers
+## @param postgres.resources.requests The requested resources for the postgres containers
+##
+postgres:
+  resources:
+    limits:
+      cpu: 700m
+      memory: 850Mi
+    requests: 
+      cpu: 400m
+      memory: 700Mi
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3.medium
+
+## cache resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param cache.resources.limits The resources limits for the cache containers
+## @param cache.resources.requests The requested resources for the cache containers
+##
+cache:
```

Comment:
for postgres and cache verticle -
limit 
 -  cpu  : 400
 -  memory : 600
 Request:
- cpu : 200
memory: 400